src/planscape/stands/models.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.gis.db import models
+
+from core.models import CreatedAtMixin
+from conditions.models import Condition
+
+
+class StandSizeChoices(models.TextChoices):
+    SMALL = "FINE", "Fine"
```

Comment:
Could you say why SMALL is getting "FINE" instead of "SMALL"?  I believe the PRD is referencing S/M/L.